test/NuGet.Core.Tests/NuGet.Common.Test/SynchronizationTest.cs

```diff
@@ -149,23 +149,6 @@ public async Task ConcurrencyUtilityDoesntBlocksInProc()
             Assert.False(timeout.IsCancellationRequested);
         }
 
-        private async Task WaitForLockToEngage(SyncdRunResult result)
```

Comment:
Are these methods not used anywhere? The removal of unused methods should be noted in the PR description.

Reply:
no, they're not used, which saved me from spending time figuring out how to annotate them and fix compiler errors, because from memory they were non-trivial to annotate. I was very happy when I noticed VS tell me 0 references!